### DISCUSSION TITLE
Options API documentation

### DISCUSSION THREAD

**Discussion by mrleblanc101** [nuxt/nuxt#16099] [Ideas]
Nuxt 3 documentation seems to only care about `<script setupt>` (Composition API).
As far as I know, the Options API is still supported in Vue 3.
Would it be possible to add more details about the Options API to the documentation.
Similar to how the Vue documentation has a global toggle to switch between the 2 mode.
Currently, I have no idea how to useLazyFetch and useLazyAsyncData work in the Options API.
![Capture d’écran, le 2022-08-25 à 12 42 02](https://user-images.githubusercontent.com/17392251/186722387-ffd18778-aee5-43e0-9594-51bf65de5f90.png)

**Comment by mlbiche** [comment]
I share the same need. Especially for `useLazyFetch` and `useLazyAsyncData`

EDIT: [Here](https://v3.nuxtjs.org/getting-started/data-fetching#options-api-support) is documentation for `asyncData`, but is there a way to have the pending and error state?

**Reply by mlbiche** [reply]
Related discussion: nuxt/framework#1681

**Comment by vedmant** [comment]
I struggle with this also when migrating my Nuxt 3 app and trying to keep options API, furthermore Nuxt 3 doesn't fully support options API, for example https://github.com/nuxt/framework/issues/9717 they don't even provide any working way to trigger error from asyncData(), basically no way to show 404 page.
